x-pack/plugins/security_solution/public/common/components/grouping/styles.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiContextMenu } from '@elastic/eui';
+import { euiStyled } from '@kbn/kibana-react-plugin/common';
+import styled from 'styled-components';
+
+export const GroupsUnitCount = styled.span`
+  font-size: ${({ theme }) => theme.eui.euiFontSizeXS};
+  font-weight: ${({ theme }) => theme.eui.euiFontWeightSemiBold};
+  border-right: ${({ theme }) => theme.eui.euiBorderThin};
+  margin-right: ${({ theme }) => theme.eui.euiSizeS};
+  padding-right: ${({ theme }) => theme.eui.euiSizeM};
+`;
+
+export const StatsContainer = styled.span`
+  font-size: ${({ theme }) => theme.eui.euiFontSizeXS};
+  font-weight: ${({ theme }) => theme.eui.euiFontWeightSemiBold};
+  border-right: ${({ theme }) => theme.eui.euiBorderThin};
+  margin-right: ${({ theme }) => theme.eui.euiSizeS};
+  padding-right: ${({ theme }) => theme.eui.euiSizeM};
+  .smallDot {
+    width: 3px !important;
+    display: inline-block;
+  }
+  .euiBadge__text {
+    text-align: center;
+    width: 100%;
+  }
+`;
+
+export const GroupingStyledContainer = styled.div`
+  .euiAccordion__childWrapper {
+    border-bottom: 1px solid #d3dae6;
```

Comment:
are these custom colors or eui theme colors? Can they be pulled from `theme.eui` as well?

Reply:
+1 using `theme.eui`, that should fix the different border colors in dark mode as well?

<img width="424" alt="Alerts_-_Kibana" src="https://user-images.githubusercontent.com/2946766/216451164-ae60de7b-d61b-428b-bbe1-3b0f88f225e8.png">